controllers/object/SplatTask.js

```diff
@@ -143,12 +158,7 @@ class SplatTask {
     async download() {
         const splatPath = path.join(objectsPath, this.object.name, identityFolderName, 'target', 'target.splat');
         try {
-            let res = await fetch(`http://${SPLAT_HOST}/downloads/${this.gaussianSplatRequestId}`, {
-                method: 'GET',
-                headers: {
-                    Authorization: `Bearer ${this.userJWT}`
-                }
-            });
+            let res = await fetch(`http://${SPLAT_HOST}/downloads/${this.gaussianSplatRequestId}`);
```

Comment:
Does the GET not require authorization? That feels like another area for future improvement (as well as the more fundamental switching SPLAT_HOST to https)

Reply:
I've only added authorization to the POST route since that's the expensive route, but I can add the auth-checking middleware to the other routes as well.

Reply:
I checked with Dan – doesn't require authorization. I had added it as I thought it fixed a previous problem, then realized that my problem was just turning zscaler on/off and that this authorization wasn't needed

Reply:
Yeah, definitely a nice-to-have in the future but not critical in the short term